### PR TITLE
Can only exec ps_trace.delete_all_traces() when Promscale not running

### PR DESCRIPTION
**Pull request by JamesGuthrie**
# Description

In https://github.com/timescale/promscale_extension/pull/437 we adjusted how this function behaves.

# Links

Related to https://github.com/timescale/promscale_extension/pull/437
Fixes https://github.com/timescale/docs/issues/1332

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [x] Is the content technically accurate?
- [x] Is the content complete?
- [x] Is the content presented in a logical order?
- [x] Does the content use appropriate names for features and products?
- [x] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
Hi @JamesGuthrie Can we please address this issue as well in the updates that you are making:https://github.com/timescale/docs/issues/1332

**Comment by JamesGuthrie** [comment]
@Rajakavitha1 this doc change and the linked change (https://github.com/timescale/promscale_extension/pull/437) address the issue raised in https://github.com/timescale/docs/issues/1332.

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 this doc change and the linked change ([timescale/promscale_extension#437](https://github.com/timescale/promscale_extension/pull/437)) address the issue raised in #1332.

thank you @JamesGuthrie

**Comment by Rajakavitha1** [comment]
@JamesGuthrie let us know when it is ready for review and we can start the doc review.

**Comment by JamesGuthrie** [comment]
@Rajakavitha1 it's ready for review!

File: promscale/manage-data/delete-data.md [hunk 1]
```diff
@@ -121,7 +121,11 @@ SELECT compress_chunk(show_chunks('prom_data.container_cpu_load_average_10s', ol
 You can delete all trace data from the database using the
 `ps_trace.delete_all_traces()` function. This function restores the schema to a
 default state, truncates the tables in the `_ps_trace` schema, and deletes all
-the data.
+the data. You can only run the function when the Promscale Connector is not
+running.
+Note: To run this function, stop the Promscale Connector, connect to the
+database and run `SELECT ps_trace.delete_all_traces;`, then start the Promscale
+Connector.
```
Comment (Rajakavitha1) [inline promscale/manage-data/delete-data.md:128]: the data. You can run the function only when the Promscale Connector is not
running.

<highlight type="note">
To run this function, stop the Promscale Connector, connect to the database and
run `SELECT ps_trace.delete_all_traces;`, then start the Promscale Connector.
</highlight>

**Review by Rajakavitha1 — APPROVED** [review]

**Pull request merged**